blenderproc/python/types/LightUtility.py

```diff
@@ -62,6 +63,96 @@ def set_type(self, type: str, frame: int = None):
         self.blender_obj.data.type = type
         Utility.insert_keyframe(self.blender_obj.data, "type", frame)
 
+    def setup_as_projector(self, fov: float, ratio: float, pattern: int, frame: int = None):
+
+        focal_length = 2 * np.tan(fov / 2)
```

Comment:
Can we not extract the fov from the current camera?

Reply:
Can we also rename ratio, to better describe what kind of ratio? 

Docu is also missing.

Reply:
pattern is a np.ndarray not a int, if I see it correctly.